src/plugin.js

```diff
@@ -26,10 +26,11 @@
   // now an array. This restores the old behavior of that property (while
   // retaining array behavior).
   Component.prototype.options = function() {
-    var options = originals.options.apply(this, arguments);
+    var component = this;
+    var options = originals.options.apply(component, arguments);
     if (Array.isArray(options.children)) {
       options.children.forEach(function(childName) {
-        options.children[childName] = this.getChild(childName).options_;
+        options.children[childName] = component.getChild(childName).options_;
```

Comment:
Hmm, what about passing `this` to `forEach` instead of this closure approach? Just a thought, not a strong opinion. :)